### PR TITLE
Structure enhancements

### PR DESCRIPTION
**Pull request by nitrogenous**
Structures Updated
- Checkbox
- ToggleButton
- RadioButton
- InputSwitch
- TriStateCheckbox

Affected Components;

- [x] Checkbox
- [x] ToggleButton
- [x] RadioButton
- [x] InputSwitch
- [x] TriStateCheckbox

Changes;
## Checkbox
_Removed_
- `hiddenInputWrapper` and `hiddenInput` from `pt` options
- `click` `focus` and `blur` from emits
- `focused` state

_Added_
- `box` to `pt` options

## RadioButton
_Removed_
- `hiddenInputWrapper` and `hiddenInput` from `pt` options
- `click` `focus` and `blur` from emits
- `focused` state

_Added_
- `box` to `pt` options

## TriStateCheckbox
_Removed_
- `hiddenInputWrapper`, `hiddenInput` and `checkbox` from `pt` options
- `click` `focus` and `blur` from emits
- `focused`state

_Added_
- `box` and `input` to `pt` options
- `readonly` to props

## InputSwitch
_Removed_
- `hiddenInputWrapper` and `hiddenInput` from `pt` options
- `click` `focus` and `blur` from emits
- `focused` state

_Added_
- `input` to `pt` options

## ToggleButton
_Removed_
- `hiddenInputWrapper` and `hiddenInput` from `pt` options
- `click` `focus` and `blur` from emits
- `focused` state

_Added_
- `input` and `box` to `pt` options
- `readonly` to props

### CODE REVIEW

File: components/lib/inputswitch/InputSwitch.js [hunk 1]
```diff
@@ -101,29 +83,24 @@ export const InputSwitch = React.memo(
             otherProps,
             ptm('root')
```
Comment (melloware) [inline components/lib/inputswitch/InputSwitch.js:84]: hmmm but won't PTM need the focused state as well?

File: components/lib/inputswitch/InputSwitch.js [hunk 2]
```diff
@@ -11,31 +11,16 @@ export const InputSwitch = React.memo(
         const mergeProps = useMergeProps();
         const context = React.useContext(PrimeReactContext);
         const props = InputSwitchBase.getProps(inProps, context);
-        const [focusedState, setFocusedState] = React.useState(false);
         const { ptm, cx, isUnstyled } = InputSwitchBase.setMetaData({
-            props,
-            state: {
-                focused: focusedState
```
Comment (melloware) [inline components/lib/inputswitch/InputSwitch.js:18]: Wasn't this here for tailwind PTM unstyled mode?

**Comment by nitrogenous** [comment]
@melloware We are updating our structures to be just like primevue for tailwind primereact so we don't need focused stated and some other things.

**Comment by melloware** [comment]
Awesome!

**Pull request merged**